jabgui/src/main/java/org/jabref/gui/walkthrough/WalkthroughAction.java

```diff
@@ -29,8 +29,11 @@
 import com.airhacks.afterburner.injection.Injector;
 
 public class WalkthroughAction extends SimpleCommand {
-    private static final Map<String, Function<Stage, Walkthrough>> WALKTHROUGH_REGISTRY = Map.of("mainFileDirectory", WalkthroughAction::createMainFileDirectoryWalkthrough);
-    private static final Map<String, Walkthrough> WALKTHROUGH_CACHE = new HashMap<>(); // must be mutable to allow caching of created walkthroughs
+    private static final Map<String, Function<Stage, Walkthrough>> WALKTHROUGH_REGISTRY = Map.of(
+            "mainFileDirectory", WalkthroughAction::createMainFileDirectoryWalkthrough,
+            "customizeEntryTable", WalkthroughAction::createEntryTableWalkthrough
+    );
```

Comment:
Move to constructor.
In static part, this will be run sometime on program startup.

Also try to lazyload.